app/js/adminPage.js

```diff
@@ -74,51 +80,62 @@ function isEmailValid(email) {
-    var result = false
+    var result = false;
     existingUsers.forEach(function(user) {
         if (user.email === emailToAdd) {
             result = true
         }
-    })
+    });
 
     return result
 }
 
 document.querySelector('#addNewUserForm').addEventListener('submit', function(event) {
-    event.preventDefault()
-    let emailField = document.querySelector('#email')
-    let nameField = document.querySelector('#name')
-    let testField = document.querySelector('#test_id')
-    let errorField = document.querySelector('#error')
-    let timeField = document.querySelector('#time')
+    event.preventDefault();
+    let emailField = document.querySelector('#email');
+    let nameField = document.querySelector('#name');
+    let testField = document.querySelector('#test_id');
+    let errorField = document.querySelector('#error');
+    let timeMinutes = parseInt(document.querySelector('#user_time_minutes').value);
+    let timeSeconds = parseInt(document.querySelector('#user_time_seconds').value);
+    let timeTotal = (timeMinutes * 60) + timeSeconds;
 
     getExistingUsers().then(function(existingUsers) {
 
-        let emailIsValid = true
-        let timeIsValid = true
+        let emailIsValid = true;
+        let timeIsValid = true;
 
         if (!isEmailValid(emailField.value) || userExists(emailField.value, existingUsers)) {
-            emailIsValid = false
-            errorField.innerHTML = "Your email is not valid or already exists: Please provide a correct email"
-        }
-        if (timeField.value <=1 || timeField.value == null || isNaN(timeField.value) === true ) {
-            timeIsValid = false
-            errorField.innerHTML += 'This is not a good number!'
-        }
+            emailIsValid = false;
+            errorField.classList.remove('alert-success');
+            errorField.classList.add('alert-danger');
+            errorField.innerHTML = "Your email is not valid or already exists: Please provide a correct email";
+        } else if (timeTotal <=1 || timeTotal > 3600 ||
+            (timeMinutes > 60 || timeMinutes < 0) ||
+            (timeSeconds > 59 || timeSeconds < 0)) {
+            timeIsValid = false;
 
-        if (emailIsValid && timeIsValid) {
-            errorField.innerHTML = ''
-            var setTime = timeField.value * 60
-            saveNewUser({'name': nameField.value, 'email': emailField.value, 'test_id': testField.value, 'time': setTime}).then(function(response) {
+            errorField.classList.remove('alert-success');
+            errorField.classList.add('alert-danger');
+            errorField.innerHTML = 'Test duration must be below an hour and minutes and seconds must be between 0 and 60.';
+        } else if (emailIsValid && timeIsValid) {
+            errorField.innerHTML = '';
+            saveNewUser({'name': nameField.value, 'email': emailField.value, 'test_id': testField.value, 'time': timeTotal}).then(function(response) {
```

Comment:
Can this be split over a few lines, it's quite hard to read when it's all on one. Also, you don't need quotes around the keys of an object.
```suggestion
            saveNewUser({
                name: nameField.value,
                email: emailField.value,
                test_id: testField.value,
                time: timeTotal
            }).then(function(response) {
```